dashboards/google-anthos/anthos-clsuter-on-vmware-vm-status.json

```diff
@@ -0,0 +1,107 @@
+{
+  "displayName": "Anthos Cluster on VMware VM Status",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "CPU Readiness Per vCPU (Percentage)",
+        "xyChart": {
+          "dataSets": [
+            {
+              "timeSeriesQuery": {
+                "timeSeriesQueryLanguage": "{ fetch k8s_pod::kubernetes.io/anthos/vm_cpu_ready_seconds | align rate(1m) | add[cluster_name: if(re_full_match(resource.namespace_name, 'kube-system'), resource.cluster_name, resource.namespace_name), ]| every 1m | group_by [resource.project_id, cluster_name, resource.location, metric.node], [value_vm_cpu_ready_seconds_aggregate: aggregate(value.vm_cpu_ready_seconds)]; fetch k8s_container::kubernetes.io/anthos/kube_node_status_capacity | filter (metric.resource == 'cpu') | every 1m | add[cluster_name: resource.cluster_name ] | group_by [resource.project_id, resource.location, cluster_name, metric.node], [value_kube_node_status_capacity_cpu_cores_aggregate: aggregate(value.kube_node_status_capacity)] } | join | div | mul (100)"
+              },
+              "plotType": "LINE"
+            }
+          ],
+          "timeshiftDuration": "0s",
+          "yAxis": {
+            "label": "y1Axis",
+            "scale": "LINEAR"
+          },
+          "chartOptions": {
+            "mode": "COLOR"
+          }
+        }
+      },
+      {
+        "title": "Memory Page Fault Latency (Percentage)",
+        "xyChart": {
+          "dataSets": [
+            {
+              "timeSeriesQuery": {
+                "timeSeriesFilter": {
+                  "filter": "metric.type=\"kubernetes.io/anthos/vm_memory_access_latency\" resource.type=\"k8s_pod\"",
+                  "aggregation": {
+                    "perSeriesAligner": "ALIGN_MEAN",
+                    "crossSeriesReducer": "REDUCE_SUM",
+                    "groupByFields": [
+                      "resource.label.\"project_id\"",
+                      "resource.label.\"location\"",
+                      "resource.label.\"cluster_name\"",
```

Comment:
Our GKE dashboards use template variables to enable users to drill down the dashboard view to just a single cluster even when multiple clusters might exist in the same project or metric scope: 

https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/google-kubernetes-engine/gke-compute-resources-cluster-view.json#L3

Is this also applicable for anthos? (Feel free to consider this a follow-up or suggestion, doesn't block this PR)

Reply:
Same goes for the other dashboards -- the images like anthos-cluster-pod-status.02.png indicate that you can have a ton of pods in a single anthos cluster, and the chart is not super usable until you filter down to what you care about.

Reply:
Synced offline, will forward comment to dashboard owner